.github/workflows/ci.yml

```diff
@@ -348,7 +348,6 @@ jobs:
           kat: false
           acvp: false
           nix-shell: ${{ matrix.compiler.shell }}
-          cflags: "-O1 -std=c11" # _FORTIFY_SOURCE requires compiling with optimization
```

Comment:
The `-std=...` lines have gone missing here

Reply:
uff. Good catch. Thanks - readded.